client-v2/src/components/FrontsEdit/CollectionComponents/Collection.tsx

```diff
@@ -18,7 +18,7 @@ import {
   createSelectCollectionStageGroups,
   createSelectCollectionEditWarning,
   selectSharedState,
-  createSelectPreviouslyLiveArticlesInCollection
+  createSelectFivePreviouslyLiveArticlesInCollection
```

Comment:
This name is very specific! It'd be good to parameterise this in the selector -- if we change the count, we have to update the code and the naming, and it's possible for developers to forget

Reply:
Oh sure, can amend now